docs/pages/repo/docs/core-concepts/caching.mdx

```diff
@@ -197,11 +197,13 @@ You can control `turbo`'s caching behavior based on
 the values of environment variables:
 
 - Including environment variables in the `env` key in your `pipeline` definition will impact the cache fingerprint on a per-task or per-workspace-task basis.
+- Including any git-ignored environment files that are read by tasks in `globalDependencies`.
```

Comment:
After reading this section, I feel like this is a bit out of place. Mentioning that adding a file to `globalDependencies` in the "Alter .... Based on Env Vars" feels a bit misleading because there is no special meaning of `.env.whatever.local` file as far as Turbo is concerned. 

What about another section under h3 section at the bottom like so:

![CleanShot 2022-11-17 at 16 16 50@2x](https://user-images.githubusercontent.com/490968/202587718-1f548c7f-dfbf-4bcf-a70b-338accfbef51.png)

Reply:
Sorry for the slow response - I had a busy few days there. I think a section about dotenv makes sense, but the reason I added the bullet point here is because the example below is showing a Next.js app, which uses dotenv.